packages/suite/src/components/suite/Settings/components/DeviceBanner.tsx

```diff
@@ -26,6 +29,14 @@ const Column = styled.div`
     display: flex;
     flex-direction: column;
     justify-content: center;
+    width: 100%;
+`;
+
+const StyledP = styled(P)`
```

Comment:
nit: I would prefer some name like `Title`, but not important.

Reply:
b09d9daf4226a80fb9f52b73e4d9d9fb2362ead1